common/src/main/java/me/shedaniel/architectury/trade/SimpleTrade.java

```diff
@@ -0,0 +1,34 @@
+package me.shedaniel.architectury.trade;
+
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.npc.VillagerTrades;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.trading.MerchantOffer;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Random;
+
+public class SimpleTrade implements VillagerTrades.ItemListing {
```

Comment:
IMO this class should be documented (including why it's needed)

Reply:
Done